ui/imports/shared/status/StatusSettingsLineButton.qml

```diff
@@ -113,13 +124,19 @@ Rectangle {
 
     MouseArea {
         anchors.fill: parent
-        enabled: root.isEnabled
         hoverEnabled: true
         onEntered: root.isHovered = true
         onExited: root.isHovered = false
         onClicked: {
-            root.clicked(!root.switchChecked)
+            if (root.isEnabled) {
+                root.clicked(!root.switchChecked)
+            }
         }
         cursorShape: Qt.PointingHandCursor
+
+        StatusToolTip {
+            text: root.toolTipText
```

Comment:
maybe a short `delay: 1000` here?

Reply:
> maybe a short `delay: 1000` here?

🤔 when sending this PR, I actually checked 😄 into the entire status-desktop code for the practice for using ToolTip and those delays are set depending on the situation... There is a default 200 delay by default, I intentionally left it because no specific reason I have to put a different one. Why do you think a 1s delay is necessary please?